### PR TITLE
Fix: Single-level wildcard match empty level

### PR DESCRIPTION
**Pull request by gnought**
Fix https://github.com/moscajs/aedes/issues/373

### CODE REVIEW

**Comment by robertsLando** [comment]
I suggest to add some info on readme for the ones using custom mqemitters about that option for empty level

Daniel

**Comment by gnought** [comment]
Or we could do it better. Check custom mqemitters if they inherit base mqemitter, and set `this.mq.matchEmptyLevels = true` directly.

**Comment by robertsLando** [comment]
I would prefer changing the defaults, most users would miss that for sure

**Comment by gnought** [comment]
Do all of you mean change the `matchEmptyLevels` default to true in `mqemitter` package?

**Comment by mcollina** [comment]
I've released mqemitter v4.1.1 with the flag turned to true.

Il giorno mer 19 feb 2020 alle ore 10:19 Gnought <notifications@github.com>
ha scritto:

File: aedes.js [hunk 1]
```diff
@@ -44,7 +44,10 @@ function Aedes (opts) {
   this.counter = 0
   this.queueLimit = opts.queueLimit
   this.connectTimeout = opts.connectTimeout
-  this.mq = opts.mq || mqemitter(opts)
+  this.mq = opts.mq || mqemitter({
+    concurrency: opts.concurrency,
+    matchEmptyLevels: true // [MQTT-4.7.1-3]
```
Comment (robertsLando) [inline aedes.js]: Should this be removed? It should be automatically turned on now

Comment (gnought) [reply aedes.js]: It is safe to set it as we know it.

**Pull request merged**